apps/system/js/nfc_handover_manager.js

```diff
@@ -631,18 +609,26 @@
 
     /**
      * Check if a device is already paired and connected.
+     *
+     * BTv2 deprecate the device.connected property.
+     * connect stat should be retrieved by adapter.getConnectedDevices API
+     *
      * @param {Object} btssp BT SSP record
      * @memberof NfcHandoverManager.prototype
      */
     _checkConnected: function _checkConnected(btssp) {
-      if (!this.bluetooth.enabled) {
+      if (!Service.query('Bluetooth.isEnabled')) {
         this._onRequestConnect(btssp);
         return;
       }
       this._findPairedDevice(btssp.mac, (device) => {
-        if (!device.connected) {
-          this._onRequestConnect(btssp);
+        if (this._adapter === null) {
+          this._logVisibly('No Bluetooth Adapter');
+          return;
         }
+        this._adapter.getConnectedDevices(device).onerror = () => {
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1090799#c20 jamin said the `device.connected` API is deprecated in BTv2